### ISSUE TITLE
/home not mounted via nfs for iso and vm images

### ISSUE THREAD

**Issue by 60up** [ltsp/ltsp#138]
Unable to get /home to mount automatically when using iso or vm images.  /home seems to mount fine using chrootless method.

User login works on client; however only default XDG directories are present. (i.e. /home is not mounted.)
localhost mount (journalctl) and issuing root `mount -a` from client yields the following error:
   `mount: /home: bad option; for several filesystems (e.g. nfs, cifs) you might need a /sbin/mount.<type> helper program.`
There is no /sbin/mount.nfs.

Can temporarily fix (until reboot) with:
`ln -s /usr/lib/klibc/bin/nfsmount /sbin/mount.nfs `
`mount -a`

How do I make the temporary fix permanent, or is there an option I missed that would make the mount work as installed?
Thanks

...---
Started with new Lubuntu 20.04 installation as server.  A chrootless image of the server (Lubuntu 20.04) mounts /home properly (/sbin/mount.nfs is present).  Both vm (Xubuntu 20.04) and iso images (Lubuntu 20.04, Xubuntu 20.04, Ubuntu Mate 20.04) seem to lack an /sbin/mount.nfs link/file.

**Comment by 60up** [comment]
The ltsp.conf options you suggested enable me to mount /home via nfs when booting debian 10 and ubuntu variant iso images.  Adding `%{SERVER}` for the `POST_INIT_HOME` is an improvement, as I had hard coded my server IP in the client fstab entry to get `nfsmount ` to work.  Thank you.  

Should note that I had to change `/usr/share/klibc/bin/nfsmount` back to `/usr/lib/klibc/bin/nfsmount` to get the commands to work.

Finally, I also found that the `POST_INIT` edits break the vm images when there is an existing `/usr/sbin/mount.nfs`.  (e.g. when nfs-common IS installed on the vm).  In this case, the vm image boot fails (stops) at the `ln` command.   Adding -b to` ln` (i.e. `ln -rsb /usr/lib/klibc/...`) enables the vm image boot to complete and keeps the existing mount.nfs as a backup file.  Not ideal, but it worked to enable /home via nfs on vm and iso images.
